pkg/commands/sso/root.go

```diff
@@ -92,7 +109,12 @@ func (c *RootCommand) Exec(in io.Reader, out io.Writer) error {
 	// that another session (from a different profile) doesn't cause unexpected
 	// errors for the user flow. This forces a re-auth.
 	if c.InvokedFromProfileCreate || c.InvokedFromProfileUpdate || ForceReAuth {
+		c.Globals.AuthServer.SetParam("prompt", "login select_account")
+	}
+	if c.InvokedFromProfileSwitch {
```

Comment:
We only want to send `select_account` on profile create, all other times (including update) we assume we know the profile email and account id. So i'd change these conditionals too:

```suggestion
	if c.InvokedFromProfileCreate || ForceReAuth {
		c.Globals.AuthServer.SetParam("prompt", "login select_account")
	}
	if c.InvokedFromProfileUpdate || c.InvokedFromProfileSwitch {
```